.gitignore

```diff
@@ -32,6 +32,7 @@ programs_external/*
 #IDE
 .vscode/settings.json
 .idea/*
+temp_ilse/*
```

Comment:
Dave had removed this from a previous PR as this is not relevant to other users of junco.